soco/core.py

```diff
@@ -326,12 +325,7 @@ def __init__(self, ip_address):
         # Note: Creation of a SoCo instance should be as cheap and quick as
         # possible. Do not make any network calls here
         super().__init__()
-        # Check if ip_address is a valid IPv4 representation.
-        # Sonos does not (yet) support IPv6
-        try:
-            socket.inet_aton(ip_address)
-        except OSError as error:
-            raise ValueError("Not a valid IP address string") from error
+
```

Comment:
I believe ip address is a string at this point.  So we should validate it as either an ipv4 or ipv6. Removing the check will just allow anything to be passed in which isn't a good solution.

Reply:
Sounds good, there are regression errors in the tests reported in CI that need to be fixed (and updated to verify a v6 address changes)